src/services/transientStorage/index.ts

```diff
@@ -37,16 +37,17 @@ const stringifyAndEncode = (obj: any): string =>
 
 const universalUrl = (url: string, key: string) => {
   const subDomain = process.env.STAGE === "production" ? "www" : "dev";
```

Comment:
- [ ] Need to double-check if `process.env.STAGE` is correctly set in production and staging environment

Reply:
https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/serverless.yml#L112 https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/serverless.yml#L75 it's setting in here